### DISCUSSION TITLE
Exported CER & PFX/PEM Unusable

### DISCUSSION THREAD

**Discussion by voyager163** [polymind-inc/acmebot#288] [Q&A] [answered]
**Describe the bug**
The cer and pfx files export from keyvault is unusable. There is a error when trying to open.

**To Reproduce**
1. Go to the acmebot function to create SSL cert
2. Go to keyvault
3. Export the CER and PFX files.
4. See error when trying to open it.
5. See error when uploading to Application Gateway http settings

**Environment (please complete the following information):**
 - Certificate Type: subdomain
 - Certificate Deploy Target: Application Gateway HTTP Setting

**Additional context**
![Screenshot 2021-03-10 at 12 51 39 AM](https://user-images.githubusercontent.com/4599055/110507546-1462b980-813b-11eb-9df5-3b95d98b853f.png)
![Screenshot 2021-03-10 at 12 46 59 AM](https://user-images.githubusercontent.com/4599055/110507559-16c51380-813b-11eb-9685-e24e93814d87.png)
![Screenshot 2021-03-10 at 12 53 03 AM](https://user-images.githubusercontent.com/4599055/110507575-1a589a80-813b-11eb-959a-d40ddd4b54d3.png)

**Comment by shibayan** [comment, marked as answer]
It's not a bug, and it has nothing to do with Acmebot since it's a Key Vault behavior.

There should be a link to the documentation for App Gateway in the README. Read the documentation first.

**Reply by voyager163** [reply]
Hi Shibayan,

Sorry, I cannot find any link on that. You mention that i can export out the certificate to be used in other place but because i am not able to export, I am not able to upload the cer file into the application gateway.

Other services
The issued certificate can be downloaded from Key Vault and used elsewhere, either in Azure or outside Azure.

**Reply by shibayan** [reply]
That has nothing to do with Acmebot because the topic is how to use Key Vault. Please refer to the Key Vault documentation.

**Comment by voyager163** [comment]
Hi Shibayan,

Thanks for pointing me to the correct direction.
